datasets/glossis/glossis.py

```diff
@@ -174,21 +170,18 @@
                 file,
                 bucket,
                 gee_bucket_folder,
-                args.assetfolder[0] + "/waterlevel/" +
+                args.assetfolder + "/waterlevel/" +
                 file.replace(".tif", ""),
                 wait=False,
                 force=True,
             )
             logging.info(f"Added task {taskid}")
             taskids.append(taskid)
 
-        # Wait for all the tasks to finish
-        wait_gee_tasks(taskids)
-
-    if args.currents:
+    if "currents" in variables:
         current_tiff_filenames = fm_to_tiff(
             bucket,
-            args.prefix[0],
+            args.prefix,
             tmpdir,
             variables=["currents_u", "currents_v"],
```

Comment:
These variables are quite different from the top level one. Maybe use `datasets` on the higher level?